seed/php-sdk/unions/no-custom-config/src/Types/Types/UnionWithLiteral.php

```diff
@@ -159,9 +165,9 @@ public static function jsonDeserialize(array $data): static
                 "JSON data is missing property 'base'",
             );
         }
-        if (!(is_string($data['base']))) {
+        if (!($data['base'] === 'base')) {
             throw new Exception(
-                "Expected property 'base' in JSON data to be string, instead received " . get_debug_type($data['base']),
+                "Expected property 'base' in JSON data to be literal, instead received " . get_debug_type($data['base']),
```

Comment:
Should this error message reference the `'base'` string instead? Otherwise the user won't know what's actually wrong.

```suggestion
                "Expected property 'base' in JSON data to be 'base', instead received " . get_debug_type($data['base']),
```